src/routes/jira/jira-get-connections-backfilStatus.ts

```diff
@@ -0,0 +1,128 @@
+import { NextFunction, Request, Response } from "express";
+import { groupBy, countBy } from "lodash";
+import { RepoSyncState } from "~/src/models/reposyncstate";
+// import { Subscription, TaskStatus } from "~/src/models/subscription";
+import { TaskStatus } from "~/src/models/subscription";
+
+type subscriptionBackfillState = {
```

Comment:
nit: we start type names with capital letters, that would be `SubscriptionBackfillState`

Reply:
Nice catch! Done.